blog/user_roles.md

```diff
@@ -0,0 +1,294 @@
+# Real World Phoenix
+## User Roles
+
+Welcome back! In the last episode of this series we implemented user authentication.
+Now let's see how we can implement user authorization by implementing different user roles into our system.
+
+We used Pow! for our authentication and that library also has a guide for implementing a user roles system.
+Let's see how far that get's us and I'd like to actually take it a bit further and implement different
+sign up flows for these different user roles.
+
+But as always, we shouldn't get ahead of ourselves and make it too complicated too soon, so we'll first
+implement basic user roles. We are going to create these users in a simple manner by just adding a selection
```

Comment:
I'm not sure I understand what you're trying to say with:

> We are going to create these users in a simple manner by just adding a selection
before the registration form that will set this field for the user

Reply:
yeah that wasn't very clear fixing it